atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractTransactionManager.java

```diff
@@ -66,8 +70,11 @@
                 log.info("[{}] Retrying transaction after {} failure(s).",
                         SafeArg.of("runId", runId),
                         SafeArg.of("failureCount", failureCount), e);
+            } catch (NotInitializedException e) {
+                log.warn("Asynchronous initialization of resources is not complete. Retrying in 10 seconds.", e);
+                Uninterruptibles.sleepUninterruptibly(AsyncInitializer.sleepInterval, TimeUnit.SECONDS);
             } catch (RuntimeException e) {
-                log.warn("[{}] RuntimeException while processing transaction.", SafeArg.of("runId", runId), e);
+                log.warn("[{}] RuntimeException while processing transaction. {}", SafeArg.of("runId", runId), e);
```

Comment:
This looks like a bug - you shouldn't need the second `{}` here (exceptions are treated differently by SLF4J - see 3 lines above).

Reply:
Yep, this is an artefact left over from debugging :/